.github/workflows/android-fastlane.yml

```diff
@@ -189,14 +189,34 @@ jobs:
           MEME_ENDPOINT: ${{ inputs.meme_endpoint }}
 
       - name: Attach android apks to release
-        uses: softprops/action-gh-release@v1
+        uses: softprops/action-gh-release@v2
         if: ${{ inputs.tag  != 'main' }}
         with:
           tag_name: ${{ inputs.tag }}
           files: |
             mobile/build/app/outputs/flutter-apk/*.apk
             mobile/build/app/outputs/flutter-apk/*.apk.sha1
 
+      - name: Compute the release tag for continues ci build
+        if: ${{ inputs.tag  == 'main' }}
+        run: |
+          echo "release_tag=v`date '+%Y-%m-%d %H:%M:%S'`" >> $GITHUB_ENV
+
+      - name: Attach android apks to release
+        uses: softprops/action-gh-release@v2
+        if: ${{ inputs.tag  == 'main' }}
+        with:
+          prerelease: true
+          name: ${{ env.release_tag }}
+          tag_name: ${{ env.release_tag }}
+          target_commitish: main
+          generate_release_notes: true
```

Comment:
🔧 I don't think we need release notes for that.

Reply:
Imho they can come handy to know what was changed.

Reply:
Wouldn't this always be just one line?

Reply:
Right... it will always be the PR which was merged...